osvr/RenderKit/RenderManagerD3D.cpp

```diff
@@ -290,5 +292,120 @@ namespace renderkit {
       return true;
     }
 
+    bool RenderManagerD3D11::GetTimingInfo(size_t whichEye,
```

Comment:
Only the DirectRender implementations had this implemented.  The extended display (which this is) did not have one.  It was added.